doc/source/release-notes/Percona-Server-5.7.21-21.rst

```diff
@@ -0,0 +1,135 @@
+.. rn:: 5.7.21-21
+
+========================
+Percona Server 5.7.21-21
+========================
+
+Percona is glad to announce the release of Percona Server 5.7.21-21
+on April 23, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.21-21/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.21
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-21.html>`_
+and includes all the bug fixes in it.
+Percona Server 5.7.21-21 is now the current GA (Generally Available) release
+in the 5.7 series. 
+
+This version of |Percona Server| includes three new encryption features: `Vault
+keyring plugin <https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#id13>`_, `encryption for InnoDB general tablespaces <https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#id7>`_, and `encryption for binary log files <https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#id14>'_.
+
+All software developed by Percona is open-source and free.
+
+New Features
+============
+
+* A new variable :variable:`innodb_temp_tablespace_encrypt` is introduced to
+  turn encryption of temporary tablespace and temporary InnoDB file-per-table
+  tablespaces on/off. Bug fixed :psbug:`3821`.
+
+* A new variable :variable:`innodb_encrypt_online_alter_logs` simultaneously
+  turns on encryption of files used by InnoDB for merge sort, online DDL logs,
+  and temporary tables created by InnoDB for online DDL. Bug fixed :psbug:`3819`.
+
+* A new variable :variable:`innodb_encrypt_tables` can be set to ``ON``, making
+  |InnoDB| tables encrypted by default, to ``FORCE``, disabling creation of
+  unencrypted tables, or ``OFF``, restoring the like-before behavior. Bug fixed
+  :psbug:`1525`.
+
+* Query response time plugin now can be disabled at session level with use 
+  of a new variable :variable:`query_response_time_session_stats`.
+
+Bugs Fixed
+==========
+
+* Query response time plugin installation was able to cause server crash,
```

Comment:
"Attempting to use a partially-installed query response time plugin could have caused server crash"